src/app/lib/device/chromecast.js

```diff
@@ -88,36 +90,17 @@
             });
         },
 
-        seek: function (seconds) {
-            console.log('Chromecast: seek %s', seconds);
-            this.get('device').seek(seconds, function (err, status) {
-                if (err) {
-                    console.error('Chromecast.seek:Error', err);
-                }
-            });
-        },
-
-        seekTo: function (newCurrentTime) {
-            console.log('Chromecast: seek to %ss', newCurrentTime);
-            this.get('device').seek(newCurrentTime, function (err, status) {
-                if (err) {
-                    console.error('Chromecast.seekTo:Error', err);
-                }
-            });
-        },
-
         seekPercentage: function (percentage) {
             console.log('Chromecast: seek percentage %s%', percentage.toFixed(2));
-            var newCurrentTime = this.get('loadedMedia').duration / 100 * percentage;
-            this.seekTo(newCurrentTime.toFixed());
+            this._seek(this.get('loadedMedia').duration / 100 * percentage);
         },
 
         forward: function () {
-            this.seek(30);
+            this._seek(this.get('loadedMedia').currentTime + 30);
         },
 
         backward: function () {
-            this.seek(-30);
+            this._seek(this.get('loadedMedia').currentTime - 30);
```

Comment:
then do seek, if you don't want to implement this i'll do it but don't complain i touch your branches...